### DISCUSSION TITLE
part-ii/2017-89/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#376] [Question comments]
Part II 2017 Logic and Set Theory: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2017-89/

**Comment by artie2000** [comment]
<details>
<summary>Examples:</summary>

Finitary: $S=L$, $T=\{\bot\}$.

Non-finitary: the set of primitive propositions. Indeed, a finite set of propositions can only constrain the valuation of finitely many primitives.

</details>
<details>
<summary>Last bit</summary>

Observe that $A\cup B\vdash\bot$, so some finite subset $A'\cup B'\vdash\bot$.

Then models of $A'$ can't be models of $B$, so $A'\vDash A$. Similarly, $B'\vDash B$.

</details>
